android/lib/theming/src/main/res/values/colors.xml

```diff
@@ -8,6 +8,7 @@
     <color name="white">#FFFFFF</color>
     <color name="black">#000000</color>
     <color name="link">#3F51B5</color>
+    <color name="bottomSheetBackground">#F8F8F8</color>
 
     //Meal
```

Comment:
Is it worth changing this comment over to the `<!-- -->` style also? (I guess the // isn't standard for Android?)

Reply:
Very true! Will fix 😄 

Yeah, `//` comments on XML is definitely not a standard